EIPS/eip-2535.md

```diff
@@ -3,7 +3,7 @@ eip: 2535
 title: Diamonds
```

Comment:
As a review comment: I do not think this is a good title. I suggest to improve this before going for Last Call.

Reply:
Understood and appreciated.  Any ideas on a better title?

Reply:
@axic How about "Diamonds, Multi-Faceted Proxy Contracts"   ?

Reply:
I created a poll about the name of the standard: https://twitter.com/mudgen/status/1419840710963081222

Reply:
I created a second poll for the name of the standard here: https://twitter.com/mudgen/status/1419865781169446915